src/components/resource/edit-custom-title/resource-edit-custom-title.js

```diff
@@ -228,20 +239,17 @@ export default class ResourceEditCustomTitle extends Component {
     let {
       model,
       proxyTypes,
-      initialValues,
     } = this.props;
 
     let {
       showSelectionModal,
       resourceSelected,
       sections,
+      initialValues,
     } = this.state;
 
-    let hasInheritedProxy = model.package &&
-      model.package.proxy &&
-      model.package.proxy.id;
-
-    let visibilityMessage = model.package.visibilityData.isHidden
+    const hasInheritedProxy = get(model, 'package.proxy.id');
+    const visibilityMessage = model.package.visibilityData.isHidden
```

Comment:
Please, rename to `visibilityMessageExists`. Current name expecting that variable contains string value but not boolean

Reply:
> Please, rename to `visibilityMessageExists`. Current name expecting that variable contains string value but not boolean

This variable actually contains a message. There is a ternary operator on the next lines